Switch from floryn90/hugo to ghcr.io/gohugoio/hugo image

julianladisch opened the pull request:
Switch from floryn90/hugo to ghcr.io/gohugoio/hugo image

The site build using docker-compose fails since floryn90/hugo v0.134.3:

```
Error: error building site: failed to acquire a build lock: open /src/.hugo_build.lock: permission denied
```

Cause: The Alpine image has switched from root user to hugo user:

https://github.com/floryn90/docker-hugo/commit/b26163074781850eba0151306837c55f8a2b5b1e

Fix: Switch to offical ghcr.io/gohugoio/hugo image that doesn't have the problem.

johngtimms commented:
Neither `floryn90/hugo:ext-alpine` nor this PR's `hugomods/hugo:std-exts` work with `main` (currently fb41620). 

I found that `image: ghcr.io/gohugoio/hugo` works.

julianladisch committed:
Switch from floryn90/hugo to ghcr.io/gohugoio/hugo image

Switch from floryn90/hugo to ghcr.io/gohugoio/hugo image

The site build using docker-compose fails since floryn90/hugo v0.134.3:

```
Error: error building site: failed to acquire a build lock: open /src/.hugo_build.lock: permission denied

Cause: The Alpine image has switched from root user to hugo user:

Florin-Lungu/docker-hugo@b261630
```

Fix: Switch to offical ghcr.io/gohugoio/hugo image that doesn't have the problem.

chalin commented:
Thx, but we're juggling too many other top priority items. Closed by:

- #494
